docs/source/daml/intro/0_Intro.rst

```diff
@@ -0,0 +1,24 @@
+An Introduction to DAML
+=======================
+
+DAML is a smart contract language designed to build composable applications on an abstract :ref:`da-ledgers`. In this intrduction, you will build an asset-holding and -trading application, and in the process learn about the structure of a DAML Ledger, and how to write DAML applications that run on any DAML Ledger implementation. You will gain an overview over most important language features, how they relate to the :ref:`da-ledgers` and how to use the DAML SDK Tools to write, test, compile, package and ship your application.
```

Comment:
```suggestion
DAML is a smart contract language designed to build composable applications on an abstract :ref:`da-ledgers`. In this introduction, you will build an asset-holding and -trading application, and in the process learn about the structure of a DAML Ledger, and how to write DAML applications that run on any DAML Ledger implementation. You will gain an overview over most important language features, how they relate to the :ref:`da-ledgers` and how to use the DAML SDK Tools to write, test, compile, package and ship your application.
```

Reply:
Diff format is not that useful for such a long line (I personally prefer breaking lines at 80 chars even for teext files, for this reason); the only change is the typo in "intrduction".